src/pages/Registration/Steps/WalletRegistration/WalletSubmission.tsx

```diff
@@ -18,13 +18,7 @@ export default function WalletSubmission({
   const { isSubmitting, registerWallet } = useRegisterWallet();
   const { data } = useRegState<3>();
 
-  if (
-    !(
-      providerId === "keplr" ||
-      providerId === "keplr-wc" ||
-      providerId === "web3auth-metamask"
-    )
-  ) {
+  if (!(providerId === "web3auth-torus")) {
```

Comment:
We have a missing `try/catch` around `disconnect` call in this component (line 38), which might cause the app to bug out on the user.
Can you add it as part of this PR?